Doc/library/zipfile.rst

```diff
@@ -551,6 +551,12 @@ Path objects are traversable using the ``/`` operator or ``joinpath``.
       Added support for text and binary modes for open. Default
       mode is now text.
 
+   .. versionchanged:: 3.11.2
+      The ``encoding`` parameter can be supplied as a positional argument
+      without causing a :exc:`TypeError`. As it could in 3.9 and earlier. Code
+      needing to be compatible with unpatched 3.10 and 3.11 versions should
+      always pass ``encoding=`` as a keyword argument.
```

Comment:
Since encoding is the first positional argument, all TextIOWrapper arguments should be passed as keyword.

Reply:
yep, reworded to be more clear.